src/server/svr_recov_db.c

```diff
@@ -311,8 +313,9 @@ sched_recov_db(char *sname, pbs_sched *ps)
 	pbs_sched *psched  = NULL;
```

Comment:
looks like you don't need psched in this function!

Reply:
That can be done with ps as well right? right now as ps is assigned with NULL even if it is not allocated there. This is a memory leak.